pvlib/data/variables_style_rules.csv

```diff
@@ -27,6 +28,9 @@ temp_module;temperature of the module
 temp_air;temperature of the air
 temp_dew;dewpoint temperature
 relative_humidity;relative humidity
+wind_speed;wind speed
+wind_direction;wind direction
```

Comment:
Should we document the angle convention like we do for `solar_azimuth`, e.g. `degrees East of North`?

Reply:
I'm not opposed, but, this is primarily a list of terms and (where needed) definitions. It seems better to leave units, angle conventions and limits to the function docstrings.